tests/playwright-test/runner.spec.ts

```diff
@@ -59,3 +59,19 @@ test('it should not allow a focused test when forbid-only is used', async ({ run
   expect(result.output).toContain('--forbid-only found a focused test.');
   expect(result.output).toContain(`- tests${path.sep}focused-test.spec.js:6 > i-am-focused`);
 });
+
+test('it should not hang and report results when worker process suddenly exits', async ({ runInlineTest }) => {
+  const result = await runInlineTest({
+    'a.spec.js': `
+      const { test } = pwt;
+      test('passed1', () => {});
+      test('passed2', () => {});
+      test('failed1', () => { process.exit(0); });
+      test('failed2', () => {});
```

Comment:
I'm surprised this one is reported as failed. I feel like that would mess with my flakiness dashboard if one test can cause everything after it to be marked as failed.

Reply:
Yeah, we should think about this and maybe figure out a better strategy.